pkgs/development/tools/mold/default.nix

```diff
@@ -0,0 +1,14 @@
+{ stdenv, fetchgit, clang_12, cmake, lld_12, tbb, xxHash, zlib, openssl, git }:
+stdenv.mkDerivation {
+  pname = "mold";
+  version = "0.9.1";
+  src = fetchgit {
```

Comment:
use fetchFromGitHub instead and don't hardcode the version, but instead use the `version` variable